src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -98,6 +98,12 @@ def load_arguments(self, _):
         nargs='+',
         help='Space-separated list of escaped JSON objects that represent the key and label filters used to build an App Configuration snapshot.'
     )
+    snapshot_status_arg_type = CLIArgumentType(
+        options_list=['--status'],
+        arg_type=get_enum_type(['archived', 'failed', 'provisioning', 'ready']),
```

Comment:
Could we get these enumeration values from Python SDK?

Reply:
@zhoxing-ms I see these status values are similar to what are used in the PollingMethod classes, however, these are also directly defined as strings.